doc/sphinx/source/project/about.rst

```diff
@@ -1,108 +1,224 @@
+    :width: 733px
+    :alt: Compiling a computation
 
-How do I connect a DL training or inference model to nGraph?
--------------------------------------------------------------
+    JiT Compiling for computation
+
+Given that we have no way to predict how many other frameworks designed around 
+model, workload, or framework-specific purposes there may be, it would be  
+impossible for us to create bridges for every framework that currently exists 
+(or that will exist in the future). Thus, we provide documentation to help 
+developers and engineers work with a lower-level API that can be used to load 
+the function graph, to run Interpreter mode to help in writing "bridge code" for 
+new or novel frameworks or algorithms. 
+
+.. csv-table::
+   :header: "Framework", "Bridge Available?", "ONNX Support?"
+   :widths: 27, 10, 10
+
+   TensorFlow, Yes, Yes
+   MXNet, Yes, Yes
+   PaddlePaddle, Coming Soon, Yes
+   PyTorch, No, Yes
+   CNTK, No, Yes
+   Other, Write your own, Custom
+
+
+How do I run an inference model?
+--------------------------------
 
 Framework bridge code is *not* the only way to connect a model (function graph) 
 to nGraph's :doc:`../ops/index`. We've also built an importer for models that 
 have been exported from a framework and saved as serialized file, such as ONNX. 
 To learn how to convert such serialized files to an nGraph model, please see 
 the :doc:`../howto/import` documentation.  
 
+.. _no-lockin:
+
+Develop without lock-in
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Being able to increase training performance or reduce inference latency by 
+simply adding another device of *any* form factor -- more compute (CPU), GPU or 
+VPU processing power, custom ASIC or FPGA, or a yet-to-be invented generation of 
+NNP or accelerator -- is a key benefit for framework developers building with 
+nGraph. Our commitment to bake flexibility into our ecosystem ensures developers' 
+freedom to design user-facing APIs for various hardware deployments directly 
+into their frameworks. 
+
+.. figure:: ../graphics/develop-without-lockin.png
+
+
+The value we're offering to the developer community is empowerment: we are 
+confident that Intel® Architecture already provides the best computational 
+resources available for the breadth of ML/DL tasks. 
+
+
+.. _features:
+
+Features
+========
+
+The nGraph IR contains a combination of device-specific and non device-specific 
+optimization and compilation paths, enabling such things as:
+
+* **Fusion** -- Fuse multiple ``Ops`` to reduce memory utilization frequency. 
+* **Memory management** -- Prevent peak memory usage by intercepting a graph 
+  and 
+* **Data reuse** -- Save result and reuse for subgraphs with the same input
+* **Graph scheduling** -- Run similar subgraphs in parallel 
+* **Graph partitioning** -- Partition subgraphs to run on different devices to 
+  speed up computation.
+* :abbr:`Direct EXecution mode (DEX)` or **DEX** -- Execute kernels for the 
```

Comment:
DEX seems like an implementation detail that does not need to be called out

Reply:
I think it's worth having it here because we are targeting framework developers and data scientist to use nGraph. When we had a meeting with Google TensorFlow team, this specific issue was brought up. The TF team wanted to make sure that the users won't need to worry about long compilation time especially for inference. They were satisfied when we mentioned about DEX. Cutting the compilation time (e.g. from 3s to 100ms for ResNet50) will provide additional reason to integrate nGraph to DL framework and will boost end users to adoption.

Reply:
No change made here.